docs/source/markdown/podman-create.1.md

```diff
@@ -881,6 +886,37 @@ content label. Shared volume labels allow all containers to read/write content.
 The `Z` option tells Podman to label the content with a private unshared label.
 Only the current container can use a private volume.
 
+  `Overlay Volume Mounts`
+
+   The `:O` flag tells Podman to mount the directory from the host as a
+temporary storage using the Overlay file system. The containers
+are allowed to modify contents within the mountpoint and are stored in the
```

Comment:
`and are` -> `which are` ... I think that the "and" would related to "The container"

Reply:
Fixed